tests/unit/unit1307.c

```diff
@@ -243,7 +243,8 @@ static const struct testcase tests[] = {
   { "?*?*?.?",                  "abcdef.c",               MATCH },
   { "?*?*?.?",                  "abcdef.cd",              NOMATCH },
 
-  { "Lindmätarv",               "Lindmätarv",             MATCH },
+  /* https://codepoints.net/U+00F6 Latin Small Letter O with Diaeresis */
```

Comment:
You accidentally changed this from "small letter a with Diaeresis" but that doesn't matter for the test.

Reply:
Nice catch, fixed!